task_12/src/js/time.js

```diff
@@ -0,0 +1,34 @@
+"use strict";
+
+document.addEventListener('DOMContentLoaded', function () {
+    var clock = document.getElementsByClassName("time")[0];
+
+    function realTimeclock() {
+        var time = new Date();
+        var hours = time.getHours().toString();
```

Comment:
It can be optimizes a little; for example:

```js
var hours = ('0' + time.getHours()).slice(-2)
```

You also may even [use loop](https://github.com/kottans-frontend-2018/assignments_Letushev/blob/task_12/js/script.js) to avoid repeating. But current code is quite fine.